content/en/apps/guides/data/analytics/couch2pg-to-cht-sync-migration.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Migrating from Couch2pg to CHT Sync"
+weight: 1
+linkTitle: "Couch2pg to CHT Sync Migration"
+description: >
+  High-level instructions on how to migrate from Couch2pg to CHT Sync for data synchronization and analytics and what to consider during the migration process.
+relatedContent: >
+  core/overview/cht-sync
+  core/overview/data-flows-for-analytics/
+---
+
+This page outlines the process of migrating from [Couch2pg](https://github.com/medic/cht-couch2pg), to the new CHT Sync pipeline. One of the main changes in the new flow involves separating the syncing process from the data transformation, with DBT now handling the latter. This migration will require converting existing SQL views and tables into DBT models. One thing to note is that the schema for CHT Sync differs from Couch2pg, so the SQL views and tables will not be directly replaced by DBT models. For instructions on how to get started with DBT models refer to the [DBT Models](({{< relref "apps/guides/data/analytics/building-dbt-models" >}}))
```

Comment:
```suggestion
This page outlines guidelines for migrating from [couch2pg](https://github.com/medic/cht-couch2pg) to the data pipeline based on CHT Sync and CHT Pipeline. One of the main changes in this flow is separating the syncing process from the data transformation, with CHT Pipeline and DBT now handling the latter. This migration requires DBT models with CHT Pipeline instead of SQL views and tables. One thing to note is that the schema for CHT Sync differs from couch2pg, so DBT models will not directly replace the SQL views and tables. For instructions on how to get started with DBT models, refer to the [DBT Models](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})).
```

I removed the word `converting` to clarify that we don't expect a direct transition but a new way of writing the data models.